scripts/es-integration-test.sh

```diff
@@ -134,6 +134,7 @@ main() {
 
   bring_up_storage "${distro}" "${version}"
   STORAGE=${distro} make storage-integration-test
+  STORAGE=${distro} SPAN_STORAGE_TYPE=${distro} make jaeger-v2-storage-integration-test
```

Comment:
in badger / es tests we run v1 and v2 as separate matrix job (in parallel), but we can add that later